tests/text_processing/tickets_data/PHPBB3-10922.txt

```diff
@@ -1 +1,5 @@
-[email]user@example.org[/email][email=user@example.org]...[/email]
\ No newline at end of file
+[email]user@example.org[/email]
+[email=user@example.org]...[/email]
```

Comment:
As one would expect. `subject` and `body` are optional, they can be set in either order. `email` is required, it can be set in short form `[email=user@example.org]` or in long form `[email email="user@example.org"]` in any order, or not at all, in which case it uses the BBCode's content as email.